pkg/storage/utils.go

```diff
@@ -1,8 +1,6 @@
-package query
+package storage
```

Comment:
Why did you rename this file to `utils.go`, I find the previous name `query.go` more appropriate :)

Reply:
i put the corresponding objects (transactions or accounts) in their own files, and appended 'Query' to most of the objects/method so that it's clearer, what do you think ?

Reply:
It's better thanks :D